python/lsst/ts/atmonochromator/monochromator_csc.py

```diff
@@ -78,6 +78,7 @@ def __init__(
         self._mock_ctrl: typing.Optional[MockController] = None
 
         self.connect_task = utils.make_done_future()
+        self.mock_server = None
```

Comment:
Why not use the `mock_ctrl` attribute that is already part of the class? 

If you are going to add this, you might want to remove the old stuff...